modules/agar/src/main/ts/ephox/agar/file/PatchInputFiles.ts

```diff
@@ -29,6 +30,8 @@ const createChangeEvent = (win: Window): Event => {
   return event;
 };
 
+const HTMLInputElement = window.HTMLInputElement;
```

Comment:
Could this be moved into the `cPatchInputElement` so this reference is only setup when needed.

Reply:
Done!